.releaserc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Not needed, that's the default configuration: https://semantic-release.gitbook.io/semantic-release/usage/configuration#branches

Reply:
ty 👍